lenskit-api/src/main/java/org/lenskit/api/ItemBasedItemRecommender.java

```diff
@@ -29,62 +26,68 @@
 
 
 /**
- * The interface for recommendation based on the items only. The difference from {@link
- * ItemRecommender} is that the input is only the item or list of items instead of user specific
- * information. This interface can be used to provide a Find Similar Items / People Also Liked
- * feature.
+ * Recommends items that go with a set of reference items. This interface is distinguished from
+ * {@link ItemRecommender} in that it uses a set of reference items instead
+ * of a user as the basis for computing scores. This component can be used to provide a Find Similar Items
+ * or People Also Liked feature.
  *
  * @see ItemRecommender
  * @author <a href="http://www.grouplens.org">GroupLens Research</a>
  * @compat Public
  * @since 0.10
  */
-public interface GlobalItemRecommender {
+public interface ItemBasedItemRecommender {
     /**
      * Recommend all possible items for a basket of items using the default exclude set.
```

Comment:
This appears to be the first use of the word basket. I think if we want this word we should put it in the top level doc, otherwise I kind of like just calling it a set of reference items.